File: smartdevices-app/smartdevicesapp.model/src/main/java/de/vogler_engineering/smartdevicesapp/model/entities/dto/JobEntryDto.java
/**
 * Copyright (c) Vogler Engineering GmbH. All rights reserved.
 * Licensed under the MIT License. See LICENSE.md in the project root for license information.
 */
package de.vogler_engineering.smartdevicesapp.model.entities.dto;

import de.vogler_engineering.smartdevicesapp.model.entities.ui.UiLayout;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class JobEntryDto extends TabEntryDto {

    private UiLayout ui;

}

File: smartdevices-gateway/SmartDevicesGateway.Model/Dto/Data/JobEntryDto.cs
//
// Copyright (c) Vogler Engineering GmbH. All rights reserved.
// Licensed under the MIT License. See LICENSE.md in the project root for license information.
//
using System;
using System.Collections.Generic;
using System.Text;
using SmartDevicesGateway.Model.Jobs;
using SmartDevicesGateway.Model.Ui;

namespace SmartDevicesGateway.Model.Dto.Data
{
    public class JobEntryDto : TabEntryDto
    {
        public UiLayout Ui { get; set; }
    }
}
